### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by tate2301** [emailjs-com/emailjs-sdk#17]
I am facing this issue on React (Next.js) and emailjs-com. I am trying to replicate the example given on [EmailJS website](https://www.emailjs.com/docs/examples/reactjs/)

```
/Users/ma/work/rkcSwissNextJS/node_modules/emailjs-com/es/index.js:1
import { init } from './methods/init/init';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:355:18)
    at wrapSafe (node:internal/modules/cjs/loader:1039:15)
    at Module._compile (node:internal/modules/cjs/loader:1073:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:10)
    at Module.load (node:internal/modules/cjs/loader:989:32)
    at Function.Module._load (node:internal/modules/cjs/loader:829:14)
    at Module.require (node:internal/modules/cjs/loader:1013:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.emailjs-com (/Users/ma/work/rkcSwissNextJS/.next/server/pages/apply.js:66:18)
    at __webpack_require__ (/Users/ma/work/rkcSwissNextJS/.next/server/webpack-runtime.js:33:42)
    at eval (webpack-internal:///./pages/apply.js:19:70)
    at Object../pages/apply.js (/Users/ma/work/rkcSwissNextJS/.next/server/pages/apply.js:44:1)
    at __webpack_require__ (/Users/ma/work/rkcSwissNextJS/.next/server/webpack-runtime.js:33:42)
    at __webpack_exec__ (/Users/ma/work/rkcSwissNextJS/.next/server/pages/apply.js:186:52)
    at /Users/ma/work/rkcSwissNextJS/.next/server/pages/apply.js:187:238
    at Function.__webpack_require__.X (/Users/ma/work/rkcSwissNextJS/.next/server/webpack-runtime.js:116:21)
```

**Comment by kumar4790** [comment]
Hey, you can add "type": "module" in package.js, then this error will fix.

**Comment by xr0master** [comment]
@kumar4790 Thank you. To be clear, do we need to add the `type` on our side?

**Comment by kumar4790** [comment]
Recommendation for @tate2301 or you assign me the issue, I will set me create a pull request for the same

**Comment by tate2301** [comment]
I do think it's also a recommendation for the SDK itself. To just have it with ES6 imports out of the box

**Comment by gurugeek** [comment]
this is a big issue and a bit of a letdown. I have upgraded the service today but now I can't use it with react/nextjs because of this error  :-(

**Comment by lucasfrey** [comment]
Same error here, will probably downgrade in the meantime :/

**Comment by tate2301** [comment]
Worked on a fix here https://github.com/emailjs-com/emailjs-sdk/pull/18 by changing to `module: "umd"`

**Comment by xr0master** [comment]
After some thought, we came to the decision not to change the support for ES modules for now, since all modern browser frameworks work better with ESM.

About Next.js, this is a well-known issue - it doesn't support ESM builds yet. They have opened up a [RFC](https://github.com/vercel/next.js/discussions/22381) to support it and the [discussion of the issue](https://github.com/vercel/next.js/issues/9607).

For next.js, you can use the `next-transpile-modules` package or EmailJS SDK version 2. The main difference between versions 2 and 3 is the support for ESM. Therefore, there is not much difference.

You can also not install our SDK, but directly send requests through the REST API.

So closing, because it's not an issue with the SDK code.

**Comment by ManuC84** [comment]
I haven't been able to make it work neither downgrading to version 2 nor using next-transpile-modules. I've wasted many hours trying to make this work which is clearly far from ideal.

**Comment by xr0master** [comment]
Hello guys,
We have released version 3.2.0 which should add native support for Next.js! *(experimental)*
Let us know how it works.
